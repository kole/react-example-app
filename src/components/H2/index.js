import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const H2 = styled.div`
    font-size: 2em;
    line-height: 1.3em;
`;

const H2Component = (props) => {
    const { text } = props;
    return <H2>{text}</H2>
};

H2Component.propTypes = {
    text: PropTypes.string.isRequired
};

export default H2Component;
