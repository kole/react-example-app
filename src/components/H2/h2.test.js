/* eslint-env jest */

import React from 'react';
import renderer from 'react-test-renderer';
import H2 from './index.js';

describe('H2', () => {
    test('Snapshot', () => {
        const component = renderer
            .create(<H2 text="H2 Component Test" />)
            .toJSON();
        expect(component).toMatchSnapshot();
    });
});
