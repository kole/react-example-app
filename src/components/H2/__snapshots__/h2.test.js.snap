// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H2 Snapshot 1`] = `
<div
  className="sc-bdVaJa QUlyn"
>
  H2 Component Test
</div>
`;
